velox/type/ShortDecimal.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/type/ShortDecimal.h"
+
+namespace facebook::velox {
+std::string ShortDecimal::toString() const {
+  return std::to_string(unscaledValue_);
```

Comment:
We should remove the `to_string` implementations here since they have no use without precision and scale.

Reply:
I can mark it VELOX_NYI, this is required for compilation. There are references in BaseVector.